src/Markdig/Parsers/BlockProcessor.cs

```diff
@@ -159,7 +166,27 @@ public BlockProcessor(MarkdownDocument document, BlockParserList parsers, Markdo
         /// <summary>
         /// Gets or sets a value indicating whether to continue processing the current line.
         /// </summary>
-        private bool ContinueProcessingLine { get; set; }
+        private bool _cpl;
```

Comment:
Don't think it is necessary to use _cpl here. Keep the full name, avoid acronyms in the code.

Reply:
fixed